charts/openmetadata/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ include "OpenMetadata.fullname" . }}-networkpolicy
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "OpenMetadata.labels" . | indent 4 }}
+  {{- with .Values.service.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: openmetadata
+  podSelector:
+    matchLabels: {{- include "OpenMetadata.selectorLabels" . | nindent 6 }}
+  policyTypes:
+  - Ingress
+    # Allow inbound connections
+  ingress:
+    - ports:
+      - port: {{ .Values.service.port }}
```

Comment:
Let's explicitly set `protocol` here to `TCP`. Check this with `kubectl explain networkpolicy.spec.ingress.ports`